sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/CbdeHandler.cs

```diff
@@ -240,9 +281,27 @@ private void RunCbdeAndRaiseIssues(CompilationAnalysisContext c)
                 pProcess.StartInfo.RedirectStandardError = true;
                 pProcess.StartInfo.UseShellExecute = false;
                 pProcess.Start();
-                pProcess.WaitForExit();
+                long totalProcessorTime=0, peakPagedMemory = 0, peakWorkingSet=0;
+                while(!pProcess.WaitForExit(100))
```

Comment:
it would be better to put `100` insto a named constant like `timeOutInMilliseconds`

Reply:
indeed. There is no magic number, not even 100.